hiyapyco/__init__.py

```diff
@@ -421,11 +461,21 @@ def _deepmerge(self, a, b, context = None):
         if self.dereferenceyamlanchors:
             a = copy.deepcopy(a)
             b = copy.deepcopy(b)
-        logger.debug('deepmerge %s and %s' % (a, b,))
-        # FIXME: make None usage configurable
+        logger.debug(
```

Comment:
I do not think that this change is related to the topic of the PR and it does not improve readability ... pls revert this change